isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/serviceschicht.adoc

```diff
@@ -0,0 +1,119 @@
+= Serviceschicht
+
+include::documentation-guide::partial$licence.adoc[]
+
+// tag::serviceschicht-definition[]
+
+Die Serviceschicht stellt Services von Backends innerhalb einer Systemlandschaft zur Verfügung.
+Sie regelt die Autorisierung von Aufrufen, validiert die übergebenen Daten und ruft die entsprechenden fachlichen Operationen des Anwendungskerns auf.
+Der Aufbau der Serviceschicht sowie ihre Kommunikation mit anderen Schichten ist in <<systemarchitektur-serviceschicht,folgender Grafik>> dargestellt.
+
+[[systemarchitektur-serviceschicht]]
+.Einbettung der Serviceschicht in die Systemarchitektur
+image::software-technisch/backend/service.dn.svg[width=80%,align=center]
+
+// end::serviceschicht-definition[]
+
+[[fachkomponenten]]
+== Fachkomponenten
```

Comment:
eventuell sollte man in der Überschrift bereits klar stellen, dass es hier um die *Fachkomponenten* der Serviceschicht geht (und nicht etwa um Fachkomponenten des Anwendungskerns)

Reply:
Ich halte den Kontext der Seite für ausreichend.